docs/authentication.md

```diff
@@ -25,5 +25,26 @@ public class Example {
 
 ```
 
+## Extension app authentication
+
+In order to authenticate your extension app, you need to have a proper `app` field defined in your configuration,
+see [authentication](authentication.md) for more information. We support both RSA and certificate authentication.
+
+With a [`SymphonyBdk`](../symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java) instance,
+this will authenticate your extension app in RSA or certificate mode based on the configuration:
+
+```java
+public class Example {
+
+    public static void main(String[] args) {
+        final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));
+        AppAuthSession appAuthSession = bdk.app("apptoken"); // token is generated by your backend
+        System.out.println("Symphony token: " + appAuthSession.getSymphonyToken());
```

Comment:
Use `log.info` instead of `sout` (cf. https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/docs/activity-api.md#how-to-create-a-command-activity)